### PR TITLE
[Calling][Feature] Update release version 'AzureCommunicationUICalling', '1.3.0'

### PR DESCRIPTION
**Pull request by ShaunaSong**
## Purpose

* update the version into 1.3.0 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

**Review by Mohtasim — DISMISSED** [review]

**Comment by jpeng-ms** [comment]
please remove the the python cache file from this PR

**Review by Mohtasim — DISMISSED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Azure Communication UI Calling Release History
+## 1.3.0 (2023-04-06)
+### New Features
+- `LocalOptions(skipSetupScreen: Bool?)` to skip the setup screen and directly join the call. [642](https://github.com/Azure/communication-ui-library-ios/pull/642)
```
Comment (Mohtasim) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:4]: As far as I remember, we ( with previous iOS calling team) agreed to not include the PR in the Changelog.

File: scripts/__pycache__/composite.cpython-310.pyc
Comment (pavelprystinka) [inline scripts/__pycache__/composite.cpython-310.pyc:1]: Please delete this file

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md (hunk 1 above)
Comment (ShaunaSong) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/CHANGELOG.md:4]: Still have PR link for chat as well

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling.podspec [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-
-  spec.dependency             'AzureCommunicationCalling', '2.2.1'
-  spec.dependency             'MicrosoftFluentUI/Avatar_ios', '0.8.9'
-  spec.dependency             'MicrosoftFluentUI/BottomSheet_ios', '0.8.9'
-  spec.dependency             'MicrosoftFluentUI/Button_ios', '0.8.9'
-  spec.dependency             'MicrosoftFluentUI/PopupMenu_ios', '0.8.9'
-  spec.dependency             'MicrosoftFluentUI/ActivityIndicator_ios', '0.8.9'
-  spec.dependency             'MicrosoftFluentUI/AvatarGroup_ios', '0.8.9'
+  
+  spec.dependency             'AzureCommunicationCalling', '2.3.0-beta.1'
```
Comment (pavelprystinka) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling.podspec:19]: We can not use beta SDK for UI library GA

**Review by pavelprystinka — CHANGES_REQUESTED** [review]

File: scripts/__pycache__/composite.cpython-310.pyc
Comment (ShaunaSong) [reply scripts/__pycache__/composite.cpython-310.pyc:1]: Deleted

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling.podspec (hunk 2 above)
Comment (ShaunaSong) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/AzureCommunicationUICalling.podspec]: Change back

**Review by pavelprystinka — APPROVED** [review]

**Review by Mohtasim — APPROVED** [review]

**Pull request merged**